types/sendsay-api/request-options.d.ts

```diff
@@ -0,0 +1 @@
+export type SendsayRequestOptions = Record<string, any>;
```

Comment:
If this file doesn't exist at runtime (and it doesn't), just include these types in the file that uses them - same with the other type definition only files - thanks~

Reply:
Thanks a lot!

Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65625/commits/bda0ef3650c0ccf1102af075ab59eb68f6f6942c